runtime/plugin/gnvim.vim

```diff
@@ -4,6 +4,9 @@ endif
 
 let g:gnvim_runtime_loaded = 1
 
+" Set blinking cursor as default
+set guicursor+=a:blinkon333
```

Comment:
Gnvim should modify `guicursor` (or other options for that manner). Instead, we should provide documentation to the users which instructs them on this topic.

Reply:
That's fine by me, the reason I included this was to make it compatible with the previous implementation (blink on by default).